website/index.html

```diff
@@ -179,10 +179,11 @@
     <div class="row mb-md-0 mb-4">
       <div class="logo-holder">
         <img id="hackpsu-logo">
+        <img src = "assets/images/logo-daytime.svg">
       </div>
       <!-- Sun and Moon Graphic -->
       <div class="celestial-body">
-        <img src="assets/images/sun.svg">
+        <img src = "assets/images/sun.svg">
```

Comment:
nitpick: ```suggestion
        <img src="assets/images/sun.svg">
```
standard HTML style has no spacing in the assignment